solidity/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@keep-network/tbtc-v2",
+    "license": "MIT",
+    "dependencies": {
+      "@openzeppelin/contracts": "^3.3.0"
+    },
+    "devDependencies": {
+      "@nomiclabs/hardhat-ethers": "^2.0.2",
+      "@nomiclabs/hardhat-waffle": "^2.0.1",
+      "chai": "^4.3.4",
+      "eslint": "^7.27.0",
+      "eslint-config-keep": "github:keep-network/eslint-config-keep#0.3.0",
+      "ethereum-waffle": "^3.3.0",
+      "ethers": "^5.3.0",
+      "hardhat": "^2.3.0",
+      "hardhat-gas-reporter": "^1.0.4",
+      "prettier": "^2.3.0",
+      "prettier-plugin-solidity": "^1.0.0-beta.11 ",
+      "solhint": "^3.3.6",
+      "solhint-config-keep": "github:keep-network/solhint-config-keep",
+      "typescript": "^4.3.2"
+    },
+    "scripts": {
```

Comment:
Let's move the `scripts` section up above `dependencies` for consistency with other projects.